scripts/stubsabot.py

```diff
@@ -305,10 +305,10 @@ async def get_github_repo_info(session: aiohttp.ClientSession, stub_info: StubMe
         split_url = urllib.parse.urlsplit(stub_info.upstream_repository)
         if split_url.netloc == "github.com":
             url_path = split_url.path.strip("/")
-            assert len(Path(url_path).parts) == 2
+            assert len(Path(url_path).parts) == 2  # noqa: PLR2004 # astral-sh/ruff#10009
```

Comment:
Hmm -- considering how many `noqa`s you've had to add for this rule, I think I'd prefer to have it just globally disabled. But please don't get rid of the changes you made to use `HTTPStatus.OK` rather than `200` -- those are great changes, even if we don't enable the rule in CI!

Reply:
That's what I was thinking on doing as well. I might revisit if https://github.com/astral-sh/ruff/issues/10009 is approved and solved. But for now ignoring it is.